desktop/build.gradle.kts

```diff
@@ -1,54 +1,36 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
 plugins {
+    application
```

Comment:
what is the purpose of the `application` plugin? Currently I don't have it and it seems to work but I did not try to create  a jar file. Maybe it is needed for that?

Reply:
The application plugin is there to make your application runnable without it you won't be able to specify the main class so the application won't run